src/moves/FreeScroll.ts

```diff
@@ -65,13 +89,13 @@ class FreeScroll extends Snap {
     // Ref #191(https://github.com/naver/egjs-flicking/issues/191)
     const lastHangerPosition = lastPosition + viewport.getRelativeHangerPosition();
 
+    const scrollAreaSize = viewport.getScrollAreaSize();
     let minimumDistanceToChange = isNextDirection
       ? currentPanelPosition + currentPanel.getSize() - lastHangerPosition + halfGap
       : lastHangerPosition - currentPanelPosition + halfGap;
+    minimumDistanceToChange = Math.abs(minimumDistanceToChange % scrollAreaSize);
```

Comment:
@WoodNeck Why did you get the rest of it?

Reply:
In scroll boundary area in circular mode, `minimumDistanceToChange` can be calculated from cloned panel 0 to original panel 0 or vice versa.
In that case, distance from original panel 0 to cloned panel 0 is added to `minimumDistanceToChange` and `scrollAreaSize` is it.
So by taking modulus, we can calculate correct `minimumDistanceToChange`